narou.gemspec

```diff
@@ -50,9 +50,9 @@ Gem::Specification.new do |gem|
 
   gem.required_ruby_version = ">=2.1.0"
 
-  gem.files         = `git ls-files`.split("\n") << Narou.create_git_commit_version
-  gem.executables   = gem.files.grep(%r{^bin/}).map{ |f| File.basename(f) }
-  gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
+  gem.files = `git ls-files`.split("\n").reject { |fn| fn =~ %r!^spec/! } << Narou.create_git_commit_version
+  gem.executables = gem.files.grep(%r{^bin/}).map{ |f| File.basename(f) }
```

Comment:
__[rubocop]__
* Style/PercentLiteralDelimiters: `%r`-literals should be delimited by `!` and `!`. [:ref](https://github.com/bbatsov/ruby-style-guide#percent-literal-braces)
* Style/SpaceBeforeBlockBraces: Space missing to the left of {.